### ISSUE TITLE
Build Fails Due to Undefined Maintainer 'nyanbinary'

### ISSUE THREAD

**Issue by god464** [lilyinstarlight/nixos-cosmic#360]
Log:
```text


error:
       … while calling the 'head' builtin

         at /nix/store/y6205wq8hxvpqvl8l9d1n9xah01kg0lq-source/lib/attrsets.nix:1575:11:

         1574|         || pred here (elemAt values 1) (head values) then
         1575|           head values
             |           ^
         1576|         else

       … while evaluating the attribute 'value'

         at /nix/store/y6205wq8hxvpqvl8l9d1n9xah01kg0lq-source/lib/modules.nix:821:9:

          820|     in warnDeprecation opt //
          821|       { value = addErrorContext "while evaluating the option `${showOption loc}':" value;
             |         ^
          822|         inherit (res.defsFinal') highestPrio;

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: undefined variable 'nyanbinary'

       at /nix/store/h8l529zqyl5v632xy45ihllqzcq25sfi-source/pkgs/cosmic-greeter/package.nix:84:39:

           83|     license = licenses.gpl3Only;
           84|     maintainers = with maintainers; [ nyanbinary /*lilyinstarlight*/ ];
             |                                       ^
           85|     platforms = platforms.linux;
builder> failed running package evaluation. You should be able to reproduce the error locally by running: prlimit --as=8589934592 nix build .#nixosConfigurations.builder.config.system.build.toplevel --dry-run --json
```
This error occurs because nyanbinary is referenced as a maintainer, but he/she appear to have left Nixpkgs, which causes the build to break.

**Comment by drakon64** [comment]
They didn't leave, just changed their name https://github.com/NixOS/nixpkgs/pull/343091

**Comment by lilyinstarlight** [comment]
Yep I saw last night and fixed this in 1599a589f0d1db5ff88c267ed59b8b77ee0b9077
